### ISSUE TITLE
Incorrect bomb position

### ISSUE THREAD

**Issue by DanielVyskoc** [saul/demofile#62]
Hello. I want to track position of bomb. This is my code:
```
demo.gameEvents.on('bomb_dropped', function(e) {
    console.log(demo.entities.entities[e.entindex].position);
});
```
However I am getting incorrect values. Output looks like this:
```
...
{ x: 27.5625,  y: 0.40625,  z: 29.4375 }
{ x: 26.125,   y: 9.4375,   z: 29.96875 }
{ x: 29.65625, y: 6.84375,  z: 16.1875 }
{ x: 21.71875, y: 29.5,     z: 21.1875 }
{ x: 13.1875,  y: 12.59375, z: 12.09375 }
{ x: 15.09375, y: 2.75,     z: 20.15625 }
{ x: 13.875,   y: 23.90625, z: 20.96875 }
{ x: 6.40625,  y: 17.59375, z: 19.15625 }
{ x: 2.96875,  y: 19.3125,  z: 15.75 }
{ x: 5,        y: 29.75,    z: 15.65625 }
...
```
Notice all values are less than 30.  Range of coordinates should be much bigger. Now it looks like bomb is always dropped very close to the center of map, but I'm sure this is not the case.

**Comment by saul** [comment]
Where is the position of the bomb on the next tick out of interest? I wonder if the event is fired before the bomb is placed in the correct location

**Comment by DanielVyskoc** [comment]
Bomb moved little bit on the next tick. I observed bomb is at [0,0,0] when picked up. When dropped coordinates equal to random number between 0 and 32. I also tried to track bomb position every second. It was always [0,0,0] or values from 0 to 32.

It looks like bomb position is relative to position of it's owner (or position of owners death). I don't know how weapon dropping in counter-strike works. Player width is 32 units. Bomb will probably drop around player randomly in circle with diameter 32, However if bomb position is relative to player pos, then it is also shifted little bit, because value is always positive. It is not in range between -16 and 16.

I also checked other weapons and they have these weird positions too.
